test/tracerEq/test_bcs_2d.py

```diff
@@ -0,0 +1,161 @@
+from thetis import *
+import pytest
+import numpy as np
+
+
+def fourier_series_solution(mesh, time):
+    """
+    Consider a diffusion problem with a inhomogeneous Neumann condition and zero initial condition:
+       c_t = nu * c_xx, c_x(0, t) = diff_flux, c_x(l, t) = 0, c(x, 0) = 0
```

Comment:
If you wish, you could use the math environment in the docstrings for better rendering:

```
Coefficient :math:`\alpha`:

.. math::
    \alpha = ...
```